### PR TITLE
Reduce usage of ternaries in Theme file

### PR DESCRIPTION
**Pull request by jarolrod**
Reduces usage of ternaries in Theme.qml by abstracting definitions of colors and image urls into "Sets" and using the definitions from the appropriate set for the current dark mode setting. 

This is a late follow-up to this review by **promag**: https://github.com/bitcoin-core/gui-qml/pull/109#discussion_r781032431

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/168)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/168)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/168)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/168)

### CODE REVIEW

**Review by shaavan — APPROVED** [review]
ACK 9ff8924c4ec6b0098b8a3bfaba745d248da579e0

1. Verified that the colorSet is correctly created with the correct values for light and dark themes.
2. Verified the proper working of the PR on Ubuntu 22.04

File: src/qml/controls/Theme.qml [hunk 1]
```diff
@@ -1,31 +1,91 @@
...
+        blocktime: "image://images/blocktime-dark"
+        network: "image://images/network-dark"
+    }
+
+    ImageSet {
+        id: lightImageSet
+        blocktime: "image://images/blocktime-light"
+        network: "image://images/network-light"
     }
+
```
Comment (hebasto) [inline src/qml/controls/Theme.qml:88]: `./test/lint/lint-whitespace.py` complains about trailing spaces:
```suggestion

```

**Comment by jarolrod** [comment]
updated from [9ff8924 ](https://github.com/bitcoin-core/gui-qml/commit/9ff8924c4ec6b0098b8a3bfaba745d248da579e0)to [6e0ee80 ](https://github.com/bitcoin-core/gui-qml/commit/6e0ee8003a45b430a0df6994522850e45c8f7894)[Compare](https://github.com/bitcoin-core/gui-qml/compare/9ff8924c4ec6b0098b8a3bfaba745d248da579e0..6e0ee8003a45b430a0df6994522850e45c8f7894)

changes: fix leading whitespace issue

File: src/qml/controls/Theme.qml (hunk 1 above)
Comment (jarolrod) [reply src/qml/controls/Theme.qml]: thanks, fixed!

**Review by hebasto — APPROVED** [review]
ACK 6e0ee8003a45b430a0df6994522850e45c8f7894, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**